Cargo.toml

```diff
@@ -13,11 +13,11 @@ categories = ["asynchronous", "os", "os::unix-apis"]
 keywords = ["timerfd", "timers"]
 
 [dependencies]
-tokio-reactor = "0.1"
-mio = "0.6"
+tokio = { version = "1.0", features = ["net"] }
 timerfd = "1.0"
-futures = "0.1"
+futures = "0.3"
```

Comment:
I would change this dependency to `futures-core` as that's a less heavy dependency when all you need is `Stream`. (Everything else you use from `futures` is available in the standard library.)